docs/knobs.md

```diff
@@ -0,0 +1,100 @@
+# Percona knobs variables
+
+Percona Server for MongoDB makes several additional configuration values
+available on the command line as well as in configuration files.
+
+## cursorTimeoutMillis
+
+Command line:
+
+```
+--cursorTimeoutMS {integer}
+```
+
+Configuration File:
+
+```
+Percona:
+  cursorTimeoutMillis: {integer}
+```
+
+Default value: 600000 (Ten Minutes)
+
+Sets the duration of time after which idle query cursors will be removed from
+memory.
+
+## failIndexKeyTooLong
+
+Command Line:
+```
+--failIndexKeyTooLong {true|false}
+```
+
+Configuration File:
+```
+Percona:
+  ignoreLongIndexError: {true|false}
+```
+
+Default value: true 
+
+Older versions of MongoDB would insert and update documents even if an index
```

Comment:
@dbpercona the description should be the opposite:
Older versions of MongoDB wouldn't insert and update documents if an index key was too long. By setting `failIndexKeyTooLong` to `false` MongoDB will insert or update the documents, but won't index keys if they are too long (so index lookup won't find such documents, although full collection scan without index usage will return the document).